 Network rinha-nginx-2024q1  Creating
 Network rinha-nginx-2024q1  Created
 Container rinha-gustavofbreunig-api02-1  Creating
 Container rinha-gustavofbreunig-db-1  Creating
 Container rinha-gustavofbreunig-api01-1  Creating
 Container rinha-gustavofbreunig-api02-1  Created
 Container rinha-gustavofbreunig-api01-1  Created
 Container nginx  Creating
 Container rinha-gustavofbreunig-db-1  Created
 Container nginx  Created
Attaching to nginx, api01-1, api02-1, db-1
db-1     | The files belonging to this database system will be owned by user "postgres".
db-1     | This user must also own the server process.
db-1     | 
db-1     | The database cluster will be initialized with locale "en_US.utf8".
db-1     | The default database encoding has accordingly been set to "UTF8".
db-1     | The default text search configuration will be set to "english".
db-1     | 
db-1     | Data page checksums are disabled.
db-1     | 
db-1     | fixing permissions on existing directory /var/lib/postgresql/data ... ok
db-1     | creating subdirectories ... ok
db-1     | selecting dynamic shared memory implementation ... posix
db-1     | selecting default max_connections ... 100
db-1     | selecting default shared_buffers ... 128MB
db-1     | selecting default time zone ... Etc/UTC
db-1     | creating configuration files ... ok
nginx    | /docker-entrypoint.sh: /docker-entrypoint.d/ is not empty, will attempt to perform configuration
nginx    | /docker-entrypoint.sh: Looking for shell scripts in /docker-entrypoint.d/
nginx    | /docker-entrypoint.sh: Launching /docker-entrypoint.d/10-listen-on-ipv6-by-default.sh
nginx    | 10-listen-on-ipv6-by-default.sh: info: Getting the checksum of /etc/nginx/conf.d/default.conf
nginx    | 10-listen-on-ipv6-by-default.sh: info: Enabled listen on IPv6 in /etc/nginx/conf.d/default.conf
nginx    | /docker-entrypoint.sh: Sourcing /docker-entrypoint.d/15-local-resolvers.envsh
nginx    | /docker-entrypoint.sh: Launching /docker-entrypoint.d/20-envsubst-on-templates.sh
nginx    | /docker-entrypoint.sh: Launching /docker-entrypoint.d/30-tune-worker-processes.sh
db-1     | running bootstrap script ... ok
nginx    | /docker-entrypoint.sh: Configuration complete; ready for start up
db-1     | performing post-bootstrap initialization ... ok
db-1     | initdb: warning: enabling "trust" authentication for local connections
db-1     | initdb: hint: You can change this by editing pg_hba.conf or using the option -A, or --auth-local and --auth-host, the next time you run initdb.
db-1     | syncing data to disk ... ok
db-1     | 
db-1     | 
db-1     | Success. You can now start the database server using:
db-1     | 
db-1     |     pg_ctl -D /var/lib/postgresql/data -l logfile start
db-1     | 
db-1     | waiting for server to start....2024-03-06 22:03:22.238 UTC [60] LOG:  starting PostgreSQL 16.2 (Debian 16.2-1.pgdg120+2) on x86_64-pc-linux-gnu, compiled by gcc (Debian 12.2.0-14) 12.2.0, 64-bit
db-1     | 2024-03-06 22:03:22.239 UTC [60] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
db-1     | 2024-03-06 22:03:22.241 UTC [63] LOG:  database system was shut down at 2024-03-06 22:03:21 UTC
db-1     | 2024-03-06 22:03:22.244 UTC [60] LOG:  database system is ready to accept connections
db-1     |  done
db-1     | server started
db-1     | CREATE DATABASE
db-1     | 
db-1     | 
db-1     | /usr/local/bin/docker-entrypoint.sh: running /docker-entrypoint-initdb.d/init.sql
db-1     | SET
db-1     | SET
db-1     | SET
db-1     | SET
db-1     | SET
db-1     | SET
db-1     | SET
db-1     | SET
db-1     | SET
db-1     | SET
db-1     | SET
db-1     | CREATE TABLE
db-1     | CREATE TABLE
db-1     | CREATE INDEX
db-1     | DO
db-1     | 
db-1     | 
db-1     | waiting for server to shut down...2024-03-06 22:03:22.740 UTC [60] LOG:  received fast shutdown request
db-1     | .2024-03-06 22:03:22.740 UTC [60] LOG:  aborting any active transactions
db-1     | 2024-03-06 22:03:22.819 UTC [60] LOG:  background worker "logical replication launcher" (PID 66) exited with exit code 1
db-1     | 2024-03-06 22:03:22.819 UTC [61] LOG:  shutting down
db-1     | 2024-03-06 22:03:22.819 UTC [61] LOG:  checkpoint starting: shutdown immediate
db-1     | 2024-03-06 22:03:22.826 UTC [61] LOG:  checkpoint complete: wrote 940 buffers (7.6%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.007 s, sync=0.001 s, total=0.007 s; sync files=0, longest=0.000 s, average=0.000 s; distance=4290 kB, estimate=4290 kB; lsn=0/191B530, redo lsn=0/191B530
db-1     | 2024-03-06 22:03:22.833 UTC [60] LOG:  database system is shut down
db-1     |  done
db-1     | server stopped
db-1     | 
db-1     | PostgreSQL init process complete; ready for start up.
db-1     | 
db-1     | 2024-03-06 22:03:22.918 UTC [1] LOG:  starting PostgreSQL 16.2 (Debian 16.2-1.pgdg120+2) on x86_64-pc-linux-gnu, compiled by gcc (Debian 12.2.0-14) 12.2.0, 64-bit
db-1     | 2024-03-06 22:03:22.919 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
db-1     | 2024-03-06 22:03:22.919 UTC [1] LOG:  listening on IPv6 address "::", port 5432
db-1     | 2024-03-06 22:03:22.919 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
db-1     | 2024-03-06 22:03:22.922 UTC [78] LOG:  database system was shut down at 2024-03-06 22:03:22 UTC
db-1     | 2024-03-06 22:03:22.926 UTC [1] LOG:  database system is ready to accept connections
nginx    | 2024/03/06 22:03:23 [error] 29#29: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.144.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://192.168.144.2:8080/clientes/1/extrato", host: "localhost:9999"
nginx    | 2024/03/06 22:03:23 [error] 29#29: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.144.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://192.168.144.4:8080/clientes/1/extrato", host: "localhost:9999"
api01-1  | 
api01-1  |   .   ____          _            __ _ _
api01-1  |  /\\ / ___'_ __ _ _(_)_ __  __ _ \ \ \ \
api01-1  | ( ( )\___ | '_ | '_| | '_ \/ _` | \ \ \ \
api01-1  |  \\/  ___)| |_)| | | | | || (_| |  ) ) ) )
api01-1  |   '  |____| .__|_| |_|_| |_\__, | / / / /
api01-1  |  =========|_|==============|___/=/_/_/_/
api01-1  |  :: Spring Boot ::                (v3.2.2)
api01-1  | 
api01-1  | 2024-03-06T22:03:23.821Z  INFO 1 --- [           main] c.r.r.RinhadrivedesignApplication        : Starting RinhadrivedesignApplication v1.0.0 using Java 17.0.10 with PID 1 (/app.jar started by root in /)
api01-1  | 2024-03-06T22:03:23.824Z  INFO 1 --- [           main] c.r.r.RinhadrivedesignApplication        : No active profile set, falling back to 1 default profile: "default"
api02-1  | 
api02-1  |   .   ____          _            __ _ _
api02-1  |  /\\ / ___'_ __ _ _(_)_ __  __ _ \ \ \ \
api02-1  | ( ( )\___ | '_ | '_| | '_ \/ _` | \ \ \ \
api02-1  |  \\/  ___)| |_)| | | | | || (_| |  ) ) ) )
api02-1  |   '  |____| .__|_| |_|_| |_\__, | / / / /
api02-1  |  =========|_|==============|___/=/_/_/_/
api02-1  |  :: Spring Boot ::                (v3.2.2)
api02-1  | 
api02-1  | 2024-03-06T22:03:24.134Z  INFO 1 --- [           main] c.r.r.RinhadrivedesignApplication        : Starting RinhadrivedesignApplication v1.0.0 using Java 17.0.10 with PID 1 (/app.jar started by root in /)
api02-1  | 2024-03-06T22:03:24.139Z  INFO 1 --- [           main] c.r.r.RinhadrivedesignApplication        : No active profile set, falling back to 1 default profile: "default"
nginx    | 2024/03/06 22:03:25 [error] 29#29: *4 no live upstreams while connecting to upstream, client: 192.168.144.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://api/clientes/1/extrato", host: "localhost:9999"
nginx    | 2024/03/06 22:03:27 [error] 29#29: *5 no live upstreams while connecting to upstream, client: 192.168.144.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://api/clientes/1/extrato", host: "localhost:9999"
api01-1  | 2024-03-06T22:03:27.626Z  INFO 1 --- [           main] .s.d.r.c.RepositoryConfigurationDelegate : Bootstrapping Spring Data JPA repositories in DEFAULT mode.
api02-1  | 2024-03-06T22:03:27.937Z  INFO 1 --- [           main] .s.d.r.c.RepositoryConfigurationDelegate : Bootstrapping Spring Data JPA repositories in DEFAULT mode.
api01-1  | 2024-03-06T22:03:27.937Z  INFO 1 --- [           main] .s.d.r.c.RepositoryConfigurationDelegate : Finished Spring Data repository scanning in 298 ms. Found 2 JPA repository interfaces.
api02-1  | 2024-03-06T22:03:28.330Z  INFO 1 --- [           main] .s.d.r.c.RepositoryConfigurationDelegate : Finished Spring Data repository scanning in 305 ms. Found 2 JPA repository interfaces.
nginx    | 2024/03/06 22:03:29 [error] 29#29: *6 no live upstreams while connecting to upstream, client: 192.168.144.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://api/clientes/1/extrato", host: "localhost:9999"
api01-1  | 2024-03-06T22:03:30.534Z  INFO 1 --- [           main] o.s.b.w.embedded.tomcat.TomcatWebServer  : Tomcat initialized with port 8080 (http)
api01-1  | 2024-03-06T22:03:30.623Z  INFO 1 --- [           main] o.apache.catalina.core.StandardService   : Starting service [Tomcat]
api01-1  | 2024-03-06T22:03:30.623Z  INFO 1 --- [           main] o.apache.catalina.core.StandardEngine    : Starting Servlet engine: [Apache Tomcat/10.1.18]
api02-1  | 2024-03-06T22:03:31.045Z  INFO 1 --- [           main] o.s.b.w.embedded.tomcat.TomcatWebServer  : Tomcat initialized with port 8080 (http)
api02-1  | 2024-03-06T22:03:31.129Z  INFO 1 --- [           main] o.apache.catalina.core.StandardService   : Starting service [Tomcat]
api02-1  | 2024-03-06T22:03:31.129Z  INFO 1 --- [           main] o.apache.catalina.core.StandardEngine    : Starting Servlet engine: [Apache Tomcat/10.1.18]
api01-1  | 2024-03-06T22:03:31.139Z  INFO 1 --- [           main] o.a.c.c.C.[Tomcat].[localhost].[/]       : Initializing Spring embedded WebApplicationContext
api01-1  | 2024-03-06T22:03:31.140Z  INFO 1 --- [           main] w.s.c.ServletWebServerApplicationContext : Root WebApplicationContext: initialization completed in 7019 ms
nginx    | 2024/03/06 22:03:31 [error] 29#29: *7 no live upstreams while connecting to upstream, client: 192.168.144.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://api/clientes/1/extrato", host: "localhost:9999"
api02-1  | 2024-03-06T22:03:31.634Z  INFO 1 --- [           main] o.a.c.c.C.[Tomcat].[localhost].[/]       : Initializing Spring embedded WebApplicationContext
api02-1  | 2024-03-06T22:03:31.636Z  INFO 1 --- [           main] w.s.c.ServletWebServerApplicationContext : Root WebApplicationContext: initialization completed in 7204 ms
api01-1  | 2024-03-06T22:03:32.637Z  INFO 1 --- [           main] o.hibernate.jpa.internal.util.LogHelper  : HHH000204: Processing PersistenceUnitInfo [name: default]
api01-1  | 2024-03-06T22:03:32.937Z  INFO 1 --- [           main] org.hibernate.Version                    : HHH000412: Hibernate ORM core version 6.4.1.Final
api01-1  | 2024-03-06T22:03:33.134Z  INFO 1 --- [           main] o.h.c.internal.RegionFactoryInitiator    : HHH000026: Second-level cache disabled
nginx    | 2024/03/06 22:03:33 [error] 29#29: *8 no live upstreams while connecting to upstream, client: 192.168.144.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://api/clientes/1/extrato", host: "localhost:9999"
api02-1  | 2024-03-06T22:03:33.316Z  INFO 1 --- [           main] o.hibernate.jpa.internal.util.LogHelper  : HHH000204: Processing PersistenceUnitInfo [name: default]
api02-1  | 2024-03-06T22:03:33.622Z  INFO 1 --- [           main] org.hibernate.Version                    : HHH000412: Hibernate ORM core version 6.4.1.Final
api02-1  | 2024-03-06T22:03:33.823Z  INFO 1 --- [           main] o.h.c.internal.RegionFactoryInitiator    : HHH000026: Second-level cache disabled
api01-1  | 2024-03-06T22:03:34.227Z  INFO 1 --- [           main] o.s.o.j.p.SpringPersistenceUnitInfo      : No LoadTimeWeaver setup: ignoring JPA class transformer
api01-1  | 2024-03-06T22:03:34.419Z  INFO 1 --- [           main] com.zaxxer.hikari.HikariDataSource       : HikariPool-1 - Starting...
api02-1  | 2024-03-06T22:03:35.016Z  INFO 1 --- [           main] o.s.o.j.p.SpringPersistenceUnitInfo      : No LoadTimeWeaver setup: ignoring JPA class transformer
api02-1  | 2024-03-06T22:03:35.229Z  INFO 1 --- [           main] com.zaxxer.hikari.HikariDataSource       : HikariPool-1 - Starting...
nginx    | 2024/03/06 22:03:35 [error] 29#29: *9 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.144.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://192.168.144.4:8080/clientes/1/extrato", host: "localhost:9999"
nginx    | 2024/03/06 22:03:35 [error] 29#29: *9 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.144.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://192.168.144.2:8080/clientes/1/extrato", host: "localhost:9999"
api01-1  | 2024-03-06T22:03:35.335Z  INFO 1 --- [           main] com.zaxxer.hikari.pool.HikariPool        : HikariPool-1 - Added connection org.postgresql.jdbc.PgConnection@74eec640
api01-1  | 2024-03-06T22:03:35.337Z  INFO 1 --- [           main] com.zaxxer.hikari.HikariDataSource       : HikariPool-1 - Start completed.
api01-1  | 2024-03-06T22:03:35.521Z  WARN 1 --- [           main] org.hibernate.orm.deprecation            : HHH90000025: PostgreSQLDialect does not need to be specified explicitly using 'hibernate.dialect' (remove the property setting and it will be selected by default)
api02-1  | 2024-03-06T22:03:36.125Z  INFO 1 --- [           main] com.zaxxer.hikari.pool.HikariPool        : HikariPool-1 - Added connection org.postgresql.jdbc.PgConnection@74eec640
api02-1  | 2024-03-06T22:03:36.135Z  INFO 1 --- [           main] com.zaxxer.hikari.HikariDataSource       : HikariPool-1 - Start completed.
api02-1  | 2024-03-06T22:03:36.246Z  WARN 1 --- [           main] org.hibernate.orm.deprecation            : HHH90000025: PostgreSQLDialect does not need to be specified explicitly using 'hibernate.dialect' (remove the property setting and it will be selected by default)
nginx    | 2024/03/06 22:03:37 [error] 29#29: *12 no live upstreams while connecting to upstream, client: 192.168.144.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://api/clientes/1/extrato", host: "localhost:9999"
nginx    | 2024/03/06 22:03:39 [error] 29#29: *13 no live upstreams while connecting to upstream, client: 192.168.144.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://api/clientes/1/extrato", host: "localhost:9999"
api01-1 exited with code 137
api02-1  | 2024-03-06T22:03:40.736Z  INFO 1 --- [           main] o.h.e.t.j.p.i.JtaPlatformInitiator       : HHH000489: No JTA platform available (set 'hibernate.transaction.jta.platform' to enable JTA platform integration)
api02-1  | 2024-03-06T22:03:40.815Z  INFO 1 --- [           main] j.LocalContainerEntityManagerFactoryBean : Initialized JPA EntityManagerFactory for persistence unit 'default'
nginx    | 2024/03/06 22:03:41 [error] 29#29: *14 no live upstreams while connecting to upstream, client: 192.168.144.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://api/clientes/1/extrato", host: "localhost:9999"
api02-1  | 2024-03-06T22:03:42.423Z  WARN 1 --- [           main] JpaBaseConfiguration$JpaWebConfiguration : spring.jpa.open-in-view is enabled by default. Therefore, database queries may be performed during view rendering. Explicitly configure spring.jpa.open-in-view to disable this warning
api02-1 exited with code 137
nginx    | 2024/03/06 22:03:43 [error] 29#29: *15 no live upstreams while connecting to upstream, client: 192.168.144.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://api/clientes/1/extrato", host: "localhost:9999"
nginx    | 2024/03/06 22:03:45 [error] 29#29: *16 no live upstreams while connecting to upstream, client: 192.168.144.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://api/clientes/1/extrato", host: "localhost:9999"
nginx    | 2024/03/06 22:04:21 [error] 29#29: *17 connect() failed (113: No route to host) while connecting to upstream, client: 192.168.144.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://192.168.144.4:8080/clientes/1/extrato", host: "localhost:9999"
nginx    | 2024/03/06 22:04:40 [error] 29#29: *17 connect() failed (113: No route to host) while connecting to upstream, client: 192.168.144.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://192.168.144.2:8080/clientes/1/extrato", host: "localhost:9999"
nginx    | 2024/03/06 22:04:45 [error] 29#29: *20 connect() failed (113: No route to host) while connecting to upstream, client: 192.168.144.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://192.168.144.4:8080/clientes/1/extrato", host: "localhost:9999"
nginx    | 2024/03/06 22:04:45 [error] 29#29: *20 no live upstreams while connecting to upstream, client: 192.168.144.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://api/clientes/1/extrato", host: "localhost:9999"
nginx    | 2024/03/06 22:04:47 [error] 29#29: *22 no live upstreams while connecting to upstream, client: 192.168.144.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://api/clientes/1/extrato", host: "localhost:9999"
nginx    | 2024/03/06 22:04:49 [error] 29#29: *23 no live upstreams while connecting to upstream, client: 192.168.144.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://api/clientes/1/extrato", host: "localhost:9999"
nginx    | 2024/03/06 22:04:54 [error] 29#29: *24 connect() failed (113: No route to host) while connecting to upstream, client: 192.168.144.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://192.168.144.2:8080/clientes/1/extrato", host: "localhost:9999"
nginx    | 2024/03/06 22:04:54 [error] 29#29: *24 no live upstreams while connecting to upstream, client: 192.168.144.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://api/clientes/1/extrato", host: "localhost:9999"
nginx    | 2024/03/06 22:04:59 [error] 29#29: *26 connect() failed (113: No route to host) while connecting to upstream, client: 192.168.144.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://192.168.144.4:8080/clientes/1/extrato", host: "localhost:9999"
nginx    | 2024/03/06 22:04:59 [error] 29#29: *26 no live upstreams while connecting to upstream, client: 192.168.144.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://api/clientes/1/extrato", host: "localhost:9999"
nginx    | 2024/03/06 22:05:01 [error] 29#29: *28 no live upstreams while connecting to upstream, client: 192.168.144.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://api/clientes/1/extrato", host: "localhost:9999"
nginx    | 2024/03/06 22:05:03 [error] 29#29: *29 no live upstreams while connecting to upstream, client: 192.168.144.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://api/clientes/1/extrato", host: "localhost:9999"
db-1     | 2024-03-06 22:05:05.551 UTC [1] LOG:  received fast shutdown request
db-1     | 2024-03-06 22:05:05.551 UTC [1] LOG:  aborting any active transactions
db-1     | 2024-03-06 22:05:05.553 UTC [1] LOG:  background worker "logical replication launcher" (PID 81) exited with exit code 1
db-1     | 2024-03-06 22:05:05.553 UTC [76] LOG:  shutting down
db-1     | 2024-03-06 22:05:05.554 UTC [76] LOG:  checkpoint starting: shutdown immediate
db-1     | 2024-03-06 22:05:05.554 UTC [76] LOG:  checkpoint complete: wrote 49 buffers (0.4%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.001 s, sync=0.001 s, total=0.001 s; sync files=0, longest=0.000 s, average=0.000 s; distance=99 kB, estimate=99 kB; lsn=0/19341A8, redo lsn=0/19341A8
db-1     | 2024-03-06 22:05:05.557 UTC [1] LOG:  database system is shut down
db-1 exited with code 0
db-1 exited with code 0
nginx exited with code 0
